.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: python
+python:
+  - 3.6
+script:
+  - python tests/discord_bot_test.py
```

Comment:
Newline

Reply:
Your Travis CI issues are because you're running the test script, but it's not the top level of your project, so it doesn't know the rest of your project exists. Take a look at: https://github.com/xelaadryth/Xelabot/blob/master/.travis.yml